### PR TITLE
[sd-3783]-(fix)-Keyboard operation of header dropdowns

### PR DESCRIPTION
**Pull request by sjunaid**

### CODE REVIEW

**Comment by mugurrus** [comment]
Please rebase

**Comment by sjunaid** [comment]
rebase done.

**Comment by ioanpocol** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/metadata/metadata.js [hunk 1]
```diff
@@ -160,8 +160,62 @@ function MetadataCtrl(
...
+                                if (buttonList.length > 0) {
+                                    var focusedElem = elem.find('button:focus')[0];
+                                    var indexValue = _.findIndex(buttonList, function(chr) {
+                                        return chr === focusedElem;
+                                    });
+
+                                    if (indexValue > 1) {
+                                        buttonList[indexValue - 1].focus();
+                                    } else {
+                                        buttonList[1].focus();
```
Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: shouldn't it select first one (index 0) ?

File: client/app/scripts/superdesk-authoring/metadata/metadata.js [hunk 2]
```diff
@@ -160,8 +160,62 @@ function MetadataCtrl(
...
+                                if (buttonList.length > 0) {
+                                    var focusedElem = elem.find('button:focus')[0];
+                                    var indexValue = _.findIndex(buttonList, function(chr) {
+                                        return chr === focusedElem;
+                                    });
+
+                                    if (indexValue > 0 && indexValue < buttonList.length - 1) {
+                                        buttonList[indexValue + 1].focus();
+                                    } else {
+                                        buttonList[1].focus();
```
Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: here probably `0` as well?

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**